radio_beam/beam.py

```diff
@@ -631,11 +633,11 @@ class EllipticalTophat2DKernel(Kernel2D):
 
     _is_bool = True
 
-    def __init__(self, width, height, position_angle, support_scaling=8, **kwargs):
+    def __init__(self, width, height, position_angle, support_scaling=2, **kwargs):
 
         self._model = EllipticalDisk2D(1. / (np.pi * width * height), 0, 0,
                                        width, height, position_angle)
         self._default_size = _round_up_to_odd_integer(support_scaling *
-                                                      np.max([width, height]))
+                                                      np.sqrt(height**2 + width**2))
```

Comment:
Increasing ```support_scaling``` for the tophat kernel just increases the amount of empty space (since it has a hard edge, unlike the gaussian), which makes morphological operators more expensive to run. IMO, I think the general use case for the tophat kernel is as a structure element, so it makes sense to return the minimal shape by default.

As for the shape, there's something funky going on here... I'll post more below